src/gui/systray.cpp

```diff
@@ -190,19 +191,34 @@ void Systray::setPauseOnAllFoldersHelper(bool pause)
     }
 }
 
-bool Systray::isOpen()
+bool Systray::isWindowVisible() const
 {
-    return _isOpen;
+    return _isWindowVisible;
 }
 
-Q_INVOKABLE void Systray::setOpened()
+void Systray::setWindowVisible(bool value)
```

Comment:
A question, regarding naming conventions. Shouldn't we use `void Systray::set**Is**WindowVisible(bool value)` since we have the variable named _**is**WindowVisible not just _windowVisible?

Reply:
This is following Qt API convention there, so windowVisible / isWindowVisible / setWindowVisible for a bool property. As for the member variable I tend to keep the is as well, there's no clear convention on that though.